src/java.base/share/classes/java/lang/Double.java

```diff
@@ -205,14 +205,30 @@ public final class Double extends Number
      */
     public static final double MIN_VALUE = 0x0.0000000000001P-1022; // 4.9e-324
 
+    /**
+     * The number of bits used to represent a {@code double} value.
+     *
+     * @since 1.5
+     */
+    public static final int SIZE = 64;
+
+    /**
+     * The number of bits in the significand of a {@code double} value.
+     * This is the parameter N in section {@jls 4.2.3} of
+     * <em>The Java Language Specification</em>.
+     *
+     * @since 19
+     */
+    public static final int PRECISION = 53;
+
     /**
      * Maximum exponent a finite {@code double} variable may have.
      * It is equal to the value returned by
      * {@code Math.getExponent(Double.MAX_VALUE)}.
      *
      * @since 1.6
      */
-    public static final int MAX_EXPONENT = 1023;
+    public static final int MAX_EXPONENT = (1 << (SIZE - PRECISION - 1)) - 1;
```

Comment:
Please include the expected value as a comment; e.g. 

(1 << (SIZE - PRECISION - 1)) - 1; // 1023

Reply:
Ready